src/index.js

```diff
@@ -1,4 +1,6 @@
 const { BaseKonnector, log, errors } = require('cozy-konnector-libs')
+const fetch = require('node-fetch')
```

Comment:
why do you need `node-fetch`?

Reply:
We need to get the list of contacts from a remote URL that is not a cozy server. `cozy-client-js` only allows to fetch cozy URLs, and I think it's the same for `cozy-client`.